include/toolchain/gcc.h

```diff
@@ -411,20 +411,35 @@ do {                                                                    \
 		",%B0"                              \
```

Comment:
We should add a comment about the sign extension that is happening in GEN_ABSOLUTE_SYM

Reply:
Added comment about `GEN_ABSOLUTE_SYM()` on accepted values, which really is dependent on architecture and toolchain.